dashboard/public/pixelation/pixelation.js

```diff
@@ -37,6 +37,65 @@ function pixelationInit() {
     bitsPerPixelRange.setAttribute("disabled", "true");
     startOver.setAttribute("disabled", "true");
   }
+
+  customizeStyles();
+  initProjects();
+}
+
+function customizeStyles() {
+  if (!window.options) {
+    // Default is version 3 (all features enabled).
+    window.options = {version: '3'};
+  }
+  if (options.version == '1') {
+    $('.hide_on_v1').hide();
+  } else if (options.version == '2') {
```

Comment:
I changed this to ===. It looks like the .level files all specify a version number as a quoted string, and that options.version is set in a way that will preserve this.